wordpress-mobile/WordPress-iOS
# [Zendesk] Update SDK to version 3.0.1

Fixes #12107

This PR updates the Zendesk SDK used in the App.

## To test:
- Run the App to make sure everything works as before
- Try to make some test with Help & Support

Update release notes:
